lib/collections/addon/provider/moderation/all/route.ts

```diff
@@ -1,11 +1,37 @@
-import Store from '@ember-data/store';
+import Transition from '@ember/routing/-private/transition';
```

Comment:
A private api, hmm? This makes me nervous. I presume this is a thing we do elsewhere? `<check check>` Ooh, yeah, all over the place. Well, we'll probably have to fix it one day.

Reply:
Actually, according to [the docs](https://api.emberjs.com/ember/3.28/classes/Transition), this is public now. Can we try using the public version?